docs/docs/guides/core-types/datasets.md

```diff
@@ -3,15 +3,24 @@ import TabItem from '@theme/TabItem';
 
 # Datasets
 
-`Dataset`s enable you to collect examples for evaluation and automatically track versions for accurate comparisons. Use this to download the latest version locally with a simple API.
+Weave _datasets_ help you to organize, collect, track, and version examples for LLM application evaluation for easy comparison. You can create and interact with datasets programmatically and via the UI. 
```

Comment:
I have a suspicion that `Dataset` is a proper noun in our product and should be capitalized.

Reply:
Agree -- `Dataset` is specifically the name of an object and should be marked so!